src/io/io.h

```diff
@@ -0,0 +1,30 @@
+#ifndef INCLUDE_IO_H
```

Comment:
Perhaps this file would better be named `io_port.h` since it seems like it will be interacting with the hardware ports.

Reply:
Done.